diagnostic.md

```diff
@@ -100,11 +134,23 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH "http://localhost:4741/donuts/42" \
+    --header "Content-Type: application/json" \
+    --data '{
+      "donut": {
+        "name": "Krüller"
+      }
+    }'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function (data) {
+  return $.ajax({
+    url: "http://localhost:4741" + '/donuts/' + data.donut.id,
```

Comment:
Don't need to use `data.donut.id`, since we aren't getting data from anywhere. Although I see what you're doing if you have a `data` object defined somewhere above.